beamer/tests/config/test_read.py

```diff
@@ -61,6 +71,14 @@ def test_config_read_request_manager(tmp_path, token, deployer):
     assert chain_config.transfer_cost == 7
     assert chain_config.target_weight_ppm == 8
 
+    # check chain removal
+    request_manager.updateChain(123, 0, 0, 0)
+    config = read_config_state(rpc_file, artifact)
+
+    assert config.block > old_block
+    assert symbol not in config.request_manager.chains
+    assert len(config.request_manager.tokens) == 0
```

Comment:
This should have been
```suggestion
    assert len(config.request_manager.chains) == 0
```
right?

Reply:
it was late last night... pushed a fix